config/fcos/v1_5_exp/translate_test.go

```diff
@@ -1632,3 +1632,28 @@ func TestTranslateGrub(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Not 100% confident on this test. Probably needs a second look if anyone has the time :)